apps/dashboard/vite.config.ts

```diff
@@ -28,10 +28,15 @@ export default defineConfig(({ mode }) => {
     server: {
       port: 5173,
       proxy: {
-        '/api': {
-          target: env.VITE_DEV_API_BASE,
+        '/api/v1': {
+          target: 'https://sudosos.test.gewis.nl/api/v1',
           changeOrigin: true,
-          rewrite: (path) => path.replace(/^\/api/, ''),
+          rewrite: (path) => path.replace(/^\/api\/v1/, ''),
+        },
+        'static': {
+          target: 'https://sudosos.test.gewis.nl/static',
+          changeOrigin: true,
+          rewrite: (path) => path.replace(/^\/static/, ''),
```

Comment:
Maybe change the sudosos.test.gewis.nl to a CONST at the top of the file? less work to only have to change a singel var.